build-posix.sh

```diff
@@ -10,4 +10,9 @@ fi
 
 LIBS="$LIBS $(pkg-config --libs sdl2) -lm"
 
+if [ ${#CC} -eq "0" ]; then
+    echo "CC variable is not set up!"
+    return
```

Comment:
But will that actually fail the whole build? Maybe something like `exit 1` would be better?

Reply:
It will actually, but I agree, `exit 1` is better